common/.eslintrc.fast.json

```diff
@@ -2,22 +2,34 @@
   "parser": "@typescript-eslint/parser",
   "plugins": [
     "@typescript-eslint",
-    "no-only-tests"
+    "no-only-tests",
+    "import"
   ],
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended"
   ],
   "rules": {
+    "@typescript-eslint/array-type": ["error", { "default": "array-simple", "readonly": "generic" }],
     "@typescript-eslint/ban-ts-ignore": "off",
+    "@typescript-eslint/consistent-type-definitions": "error",
+    "@typescript-eslint/explicit-member-accessibility": [
+      "error",
+      {
+        "accessibility": "explicit",
+        "overrides": { "constructors": "no-public" }
+      }
+    ],
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/prefer-for-of": "error",
+    "@typescript-eslint/prefer-function-type": "error",
+    "@typescript-eslint/unified-signatures": "error",
```

Comment:
nit: not a fan.

Reply:
I do like this rule in theory since overloads are not really a javascript feature - and a unified signature better represents the underlying function, but I do remember encountering false positives with this rule - can be omitted IMO.